storage-manager/src/IOCoordinator.cpp

```diff
@@ -1135,7 +1168,10 @@ boost::shared_array<uint8_t> IOCoordinator::mergeJournal(const char *object, con
     
```

Comment:
The changes to mergeJournal are just little things I found while waiting for information.  Mainly, saving errno, and propagating kpis to the caller.